config/mentee_evaluation.yml

```diff
@@ -241,3 +245,13 @@ experienced:
   - answer: "They hide their real experience"
     hint: "If they state they don't know some language confidently, but you can tell from their application that they should"
     score: "1"
+ dedication:
```

Comment:
why is it defined twice? And it's a new key so we need to add it to application flow

Reply:
As there already was a distinction between experienced/not experienced I wasn't sure how you use the set of questions and added changes and new dedication key to both sections. The changes made would make it possible to just use one set of the questions/evaluation options for both, experienced and inexperienced mentee applicants. Not sure though about implications for that change.